content/en/quickstart/quickstart-ci.md

```diff
@@ -0,0 +1,195 @@
+          with:
+            inputs: to_be_signed.txt
+```
+
+When run, this workflow returns the ephemeral certificate used to sign the file, as well as the index for the transparency log entry.
+
+### Verifying your signed files
+
+The `gh-action-sigstore-python` GitHub Action includes an option to verify your generated signature. This is optional but a great way to understand the GitHub Action as you are integrating it into your CI for the first time. To verify the signature you just created, set the `verify` setting to true and include your expected `verify-cert-identity` and `verify-oidc-issuer` settings.
+
+```console
+        - uses: sigstore/gh-action-sigstore-python@v3.0.0
+          with:
+            inputs: to_be_signed.txt
+            verify: true
+            verify-cert-identity: https://github.com/USERNAME/REPOSITORY_NAME/.github/workflows/WORKFLOW_NAME@refs/heads/BRANCH_NAME
+            verify-oidc-issuer: https://token.actions.githubusercontent.com
+```
+
+## Using Cosign within your CI system
+
+If you need functionality beyond simple signing of files and blobs, you can use the [`consign-installer` GitHub Action](https://github.com/marketplace/actions/cosign-installer) to [integrate Sigstore into your CI system](#installing-cosign-on-your-ci). This quickstart covers:
+
+- How to [sign and verify a container image](#signing-and-verifying-a-container-image) using your CI system
+- How to [sign](#signing-a-blob) and [verify](#verifying-a-blob) a blob using `consign-installer`
+
+### Installing Cosign on your CI
+
+The following workflow will install Cosign into your workflow environment.
+
+```console
+name: install-cosign-and-use
+on:
+  # This will trigger the workflow to run when commits are pushed to the main branch. This is easy for testing purposes, but for your final workflow use whatever event or schedule makes sense for your project.
+   push:
+    branches: [ main ]
+# No special permissions are required to install cosign, but `id-token: write` is needed to sign with your workflow identity.
+permissions:
+  id-token: write
+
+jobs:
+  install-cosign-and-use:
+    name: Install Cosign
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Cosign
+        uses: sigstore/cosign-installer@v3.7.0
+      - name: Check install!
+        run: cosign version
+```
+
+### Signing and verifying a container image
+
+The ability to sign and verify container images is the primary benefit of using the cosign-installer GitHub Action. The following is an example workflow that will build a container image with QEMU and Docker Buildx, push that image to the GitHub Container Registry, sign the image, and then verify it. Replace your username, repository name, and branch name where indicated.
+
+```console
+name: container-signing-and-verifying
+on:
+    push:
+        branches: [ main ]
+permissions:
+    contents: read
+    packages: write
+    id-token: write # needed for signing the images with GitHub OIDC Token            
```

Comment:
The job below configures its permissions, so I think this could be removed or set to `{}` explicitly here:

```suggestion          
```